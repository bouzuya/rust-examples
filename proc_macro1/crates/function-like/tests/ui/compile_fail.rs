fn main() {
    function_like::function_like!(123);
}
