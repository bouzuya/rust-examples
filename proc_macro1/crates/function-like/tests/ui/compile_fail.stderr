error: expected string literal
 --> tests/ui/compile_fail.rs:2:35
  |
2 |     function_like::function_like!(123);
  |                                   ^^^
